sky/backends/cloud_vm_ray_backend.py

```diff
@@ -107,6 +107,13 @@
     pathlib.Path(sky.__file__).resolve().parent / 'backends' /
     'monkey_patches' / 'monkey_patch_ray_up.py')
 
+# Restart skylet when the version does not match to keep the skylet up-to-date.
+_MAYBE_SKYLET_RESTART_CMD = (
+    f'[[ $(cat {constants.SKYLET_VERSION_FILE}) = "{constants.SKYLET_VERSION}"'
+    ' ]] || (pkill -f "python3 -m sky.skylet.skylet";'
+    f' echo {constants.SKYLET_VERSION} > {constants.SKYLET_VERSION_FILE};'
+    'nohup python3 -m sky.skylet.skylet >> ~/.sky/skylet.log 2>&1 &);')
+
```

Comment:
Did we test this manually, just to make sure it does not update the skylet every time?